db/sqldb/golden/schema_dump.sql

```diff
@@ -116,7 +116,12 @@ CREATE TYPE public.file_type AS ENUM (
     'yaml',
     'zip',
     'html',
-    'json'
+    'json',
+    'txt',
+    'css',
+    'js',
+    'ttf',
+    ''
```

Comment:
Let's not add `''` as a valid enum type, we don't want to persist it.

Reply:
Discussed in the comment above, I think we do. I'm also open to calling this `unknown` or whatever you think makes sense here